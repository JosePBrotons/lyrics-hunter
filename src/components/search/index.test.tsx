import React from 'react';
import renderer from 'react-test-renderer';
import Search from '.';

test('renders correctly', () => {
    const searchCmp = renderer.create(<Search navigation={null} />).toJSON();
    expect(searchCmp).toMatchSnapshot();
});