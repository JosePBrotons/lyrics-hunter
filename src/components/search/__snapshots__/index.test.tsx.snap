// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#e9e9e9",
      "flex": 1,
      "flexDirection": "column",
      "paddingHorizontal": 16,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "paddingTop": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#F8F8F8",
          "borderRadius": 5,
          "color": "#454545",
          "flexDirection": "row",
          "fontSize": 14,
          "height": 40,
          "lineHeight": 22,
          "marginBottom": 20,
          "paddingHorizontal": 10,
          "width": "80%",
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#6A6A6A",
              "fontSize": 16,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Artist"
        placeholderTextColor="#6A6A6A"
        rejectResponderTermination={true}
        secureTextEntry={false}
        style={
          Object {
            "color": "#454545",
            "flex": 1,
            "paddingLeft": 8,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#F8F8F8",
          "borderRadius": 5,
          "color": "#454545",
          "flexDirection": "row",
          "fontSize": 14,
          "height": 40,
          "lineHeight": 22,
          "marginBottom": 20,
          "paddingHorizontal": 10,
          "width": "80%",
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#6A6A6A",
              "fontSize": 16,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Song"
        placeholderTextColor="#6A6A6A"
        rejectResponderTermination={true}
        secureTextEntry={false}
        style={
          Object {
            "color": "#454545",
            "flex": 1,
            "paddingLeft": 8,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#0682FE",
          "borderRadius": 5,
          "height": 40,
          "justifyContent": "center",
          "marginBottom": 10,
          "marginTop": 5,
          "opacity": 0.5,
          "width": "60%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 16,
            "lineHeight": 22,
          }
        }
      >
        Search
      </Text>
    </View>
  </View>
</RCTSafeAreaView>
`;
